kafkaesk/consumer.py

```diff
@@ -426,6 +431,7 @@ async def healthy(self) -> None:
     async def on_partitions_revoked(self, revoked: typing.List[aiokafka.TopicPartition]) -> None:
```

Comment:
@dmanchon when `on_partitions_revoked` is called, are we certain that no consumers are in-flight and working on processing some messages?